eeps/eep-0048.md

```diff
@@ -101,32 +101,35 @@ the type specification below:
 
     {docs_v1,
      Anno :: erl_anno:anno(),
-     Language :: atom(),
+     BeamLanguage :: atom(),
      Format :: mime_type(),
-     ModuleDoc :: binary() | none | hidden,
+     ModuleDoc :: %{DocLanguage := DocString} | none | hidden,
      Metadata :: map(),
      Docs ::
        [{{Kind, Name, Arity},
          Anno :: erl_anno:anno(),
          Signature :: [binary()],
-         Doc :: binary() | none | hidden,
+         Doc :: %{DocLanguage := DocString} | none | hidden,
          Metadata :: map()
-        }]}
+        }]} when DocLanguage :: binary(),
+                 DocString :: binary()
 
 where in the root tuple we have:
 
   * `Anno` - annotation (line, column, file) of the module documentation
-    or of the definition itself (see erl_anno)
+    or of the definition itself (see `erl_anno`)
```

Comment:
You have somehow managed to get two non-breaking-space in the beginning of this line: [32,160,32,160]